File: 003-Transaction/MS-Transaction.Domain/Entities/Transfer.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MS_Transaction.Domain.Entities
{
    [Table("Transfers", Schema = "Toro")]
    public class Transfer
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [MaxLength(8), Required]
        public string Event { get; set; }

        [Required]
        public decimal Amount { get; set; }

        public Target Target { get; set; }

        public Origin Origin { get; set; }

        public DateTime? Created { get; set; }
    }
}

File: 005-UI/src/app/dto/transfer.ts
import { Stock } from "./stock";

export class Transfer {
    event: string = "TRANSFER";
	target: Target;
	origin: Origin;
	amount: any;
}

export class Target {
    bank: string;
    branch: string; 
    account: string; 
}

export class Origin {
    bank: string;
    branch: string; 
    cpf: string; 
}